token/cli/src/main.rs

```diff
@@ -2877,14 +2923,19 @@ fn process_command(
             let token = pubkey_of_signer(arg_matches, "token", &mut wallet_manager)
                 .unwrap()
                 .unwrap();
-            let amount = value_t_or_exit!(arg_matches, "amount", f64);
+            let amount = value_t_or_exit!(arg_matches, "amount", u64);
```

Comment:
Be careful, this won't be correct if someone wants to mint `1.5` tokens.  The best way to do this is convert from the string in `amount` to a u64 amount using `spl_token::try_ui_amount_into_amount`

Reply:
Need to work on this